invenio/b2share/modules/b2deposit/b2share_model/metadata/linguistics_lang_codes.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 
 lang_codes = [
+    ('eng', 'English'),
```

Comment:
@noumar You are correct that duplicating a field is not the best option, but the previous behaviour was much worse. I don't know exactly how to modify the model converter, do you?